### PR TITLE
Web UI: rework server VNC

### PR DESCRIPTION
**Pull request by ylei-tsubame**
This PR:
* removes the SSH tunnel component from the VNC pipe; the pipe only consists of `websockify`
* reduces the number of database queries during VNC pipe setup; AFAIK, queries are only required when stopping VNC pipe
* forwards websocket connections from UI to subnodes; this should allow VNC to work if the user can load the UI (`ws://` traffic will go to the same port as `http://`); this method also greatly improves connection speed
* reconnects VNC connection automatically when it disconnects; however, it's unreliable during server migration because the `novnc` module can't reliably detect a abrupt disconnect (need more time to refine)
* Disables Apache on package `striker` update

This PR is ready to merge **after** several CI runs.

### CODE REVIEW

**Comment by fabbione** [comment]
retest this please

**Comment by ylei-tsubame** [comment]
Triggered 3rd test with rebase on top of `main`.

**Comment by fabbione** [comment]
retest this please

File: Anvil/Tools/Remote.pm [hunk 1]
```diff
@@ -477,6 +483,7 @@ sub call
 					user       => $remote_user,
 					port       => $port, 
 					batch_mode => 1,
+					@$ossh_opts,
```
Comment (digimer) [inline Anvil/Tools/Remote.pm:486]: This is technically correct, but generally we use '@{$ossh_opts}' to make the variable with the array reference easier to see.

**Review by digimer — APPROVED** [review]

**Pull request merged**